blocks/subscriptions-block/components/ActionDialog/styles.scss

```diff
@@ -0,0 +1,79 @@
+.xpmedia-action-dialog {
+  display: grid;
+  grid-template-columns: map-get($spacers, 'md') repeat(6, 1fr) map-get($spacers, 'md');
```

Comment:
The designs show a 12 column grid, any particular reason to reimplement a grid layout here?

Reply:
I can make it a full width layout, but I thought this was going to be a re-usable component and the width would be determined by the parent.  Therein, this is only 8 of 12 columns.

Reply:
Also, do we have a column css layout we are using?  I see a column definition, but no specific column rules.

Reply:
Updated to 12 column.  Adding another breakpoint because the `md` was causing some strange narrowness to some screen sizes.